src/constants/veNft/veNftProject.ts

```diff
@@ -0,0 +1,3 @@
+export const VENFT_CONTRACT_ADDRESS =
```

Comment:
**question** guessing we can eventually get this from an npm package?

Reply:
Yes. It is an outstanding issue to move the veNft stuff including ABIs into the JBContracts directory. veNft will also be launched on a per-project basis and fetched similarly to how a project's ERC-20 token is retreived.

Reply:
Partially tracked in https://github.com/jbx-protocol/juice-interface/issues/1526, launching and retrieving is part of https://github.com/jbx-protocol/juice-interface/issues/1416 which is an epic-level issue that i need to scope and will require discussions with JB contract folks.